src/app/api/auth/onboarding/route.ts

```diff
@@ -13,9 +14,22 @@ export async function POST(req: Request) {
   try {
     const owaData = onboardingSchema.parse(await req.json());
     const sourceApp = user.publicMetadata.sourceApp ?? getReferrerOrigin(req);
+    const region =
+      req.headers.get("x-country") ?? getBrowserConfig("developmentUserRegion");
```

Comment:
Do we know if Netlify will set a value if it can't geolocate a user? I'd hate for onboarding to fail and lock out someone because their IP isn't mapped in the ip=>geo database. I know that Cloudflare set `XX` when it can't locate a user, perhaps Netlify do something similar?

In case they don't set a fallback value I don't think this should block onboarding. There are still likely to be things a user can do legitimately after signing in that don't require us knowing their region. Maybe this could be logged instead?

Reply:
I couldn't find out a lot about x-country, nfortunately we cant use the cloudflare one as is seems to be filtered out by netlify. logged instead? - do you mean report to bugsnag in an Oak Error but not thrown?

Reply:
Yeah reported to BugSnag with the user id but handled, would help us get an idea of how many user's have a missing region while not blowing up their onboarding.

Reply:
I've not managed to get the errorReporter to work in the route file but no longer locks user in.